src/include/stir/recon_buildblock/BinNormalisationFromProjData.h

```diff
@@ -99,6 +99,8 @@ class BinNormalisationFromProjData :
   virtual void undo(RelatedViewgrams<float>& viewgrams,const double start_time, const double end_time) const;
 
   virtual float get_bin_efficiency(const Bin& bin,const double start_time, const double end_time) const;
+    //! Get a shared_ptr to the normalisation proj_data.
+  virtual shared_ptr<ProjData> get_norm_proj_data_sptr() const;
```

Comment:
do we really need this? It will break once we resolve #348 (or would make that very inefficient)

Reply:
For this: